assets/src/editor/events/tickets/editor-ticket/price-calculator/price-modifier-row/add-price-modifier-action-button.js

```diff
@@ -67,4 +68,13 @@ const AddPriceModifierActionButton = ( {
 	);
 };
 
+AddPriceModifierActionButton.propTypes = {
+	ticket: PropTypes.object.isRequired,
+	lastRow: PropTypes.bool,
```

Comment:
missing a couple of props here:

- lastPrice - object
- priceTypes - arrayOf( object )